cli/template/base/_eslintrc.cjs

```diff
@@ -1,25 +1,15 @@
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const path = require("path");
-
 /** @type {import("eslint").Linter.Config} */
-const config = {
-  overrides: [
-    {
-      extends: [
-        "plugin:@typescript-eslint/recommended-requiring-type-checking",
-      ],
-      files: ["*.ts", "*.tsx"],
-      parserOptions: {
-        project: path.join(__dirname, "tsconfig.json"),
-      },
-    },
-  ],
+module.exports = {
   parser: "@typescript-eslint/parser",
   parserOptions: {
-    project: path.join(__dirname, "tsconfig.json"),
+    project: true,
```

Comment:
[`project: true` is already in typescript-eslint@v5](https://typescript-eslint.io/linting/typed-linting). Just figured I'd simplify things a bit while I'm here. 😄

Reply:
Huh you're right! Great catch, TIL. 

Looking deeper, the `module.exports` property is typed as `NodeJS.Module.exports: any`. So putting a `/** @type */` is ineffective because the property is seen as `any` first. 🤷